### ISSUE TITLE
dose not work with nextjs

### ISSUE THREAD

**Issue by moislaieh** [joeattardi/picmo#223]
just by importing this line 

`import { createPicker } from "picmo"`

gives you an error , **localStorage is not defined**

**Comment by joeattardi** [comment]
Thanks for catching this.

The problem took me a while to track down but in the default options object, the local storage based recents provider is instantiated as a default provider. Because this is executed immediately, it tries to access localStorage on import.

A fix has been committed and this will be included in the next version of PicMo!

In the meantime, you may be able to get around this by using a dynamic import within a `if (process.browser)` check:

```javascript
if (process.browser) {
  const { createPicker } = await import('picmo');
}
```

The next release should be ready soon.

**Comment by joeattardi** [comment]
I just published version 5.4.1, you should be able to import PicMo now without it attempting to reference `localStorage`.
